proxy/coco/tests/gossip.rs

```diff
@@ -12,30 +12,33 @@ use common::{
     build_peer, build_peer_with_seeds, connected, init_logging, radicle_project, shia_le_pathbuf,
 };
 
-#[tokio::test]
+#[tokio::test(core_threads = 2)]
 async fn can_announce_new_project() -> Result<(), Box<dyn std::error::Error>> {
     init_logging();
 
     let alice_tmp_dir = tempfile::tempdir()?;
-    let alice_store = kv::Store::new(kv::Config::new(alice_tmp_dir.path().join("store")))?;
     let alice_repo_path = alice_tmp_dir.path().join("radicle");
-    let (alice_peer, alice_state, alice_signer) = build_peer(&alice_tmp_dir).await?;
+    let (alice_peer, alice_state, alice_signer) = build_peer(
+        &alice_tmp_dir,
+        RunConfig {
+            announce: AnnounceConfig {
+                interval: Duration::from_millis(100),
+            },
+            ..RunConfig::default()
+        },
+    )
+    .await?;
     let alice_events = alice_peer.subscribe();
 
-    tokio::task::spawn(alice_peer.run(alice_state.clone(), alice_store, RunConfig::default()));
+    let _alice_runs = alice_peer.into_running();
```

Comment:
Nb funny semantics: we don't actually need to `spawn` this, just prevent it from being dropped. It also doesn't _hurt_ to spawn it, though, so whatever people think is preferable in tests